rke/k8s_rke_system_images.go

```diff
@@ -5778,6 +5778,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled out of band post v2.5.11
+		"v1.20.13-rancher1-1": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.4.15-rancher1",
+			Kubernetes:                "rancher/hyperkube:v1.20.13-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.78",
+			NginxProxy:                "rancher/rke-tools:v0.1.78",
+			CertDownloader:            "rancher/rke-tools:v0.1.78",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.78",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.15.10",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.15.10",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.15.10",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.1",
+			Flannel:                   "rancher/mirrored-coreos-flannel:v0.15.1",
+			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher6",
+			CalicoNode:                "rancher/mirrored-calico-node:v3.17.2",
+			CalicoCNI:                 "rancher/mirrored-calico-cni:v3.17.2",
+			CalicoControllers:         "rancher/mirrored-calico-kube-controllers:v3.17.2",
+			CalicoCtl:                 "rancher/mirrored-calico-ctl:v3.17.2",
+			CalicoFlexVol:             "rancher/mirrored-calico-pod2daemon-flexvol:v3.17.2",
+			CanalNode:                 "rancher/mirrored-calico-node:v3.17.2",
+			CanalCNI:                  "rancher/mirrored-calico-cni:v3.17.2",
+			CanalControllers:          "rancher/mirrored-calico-kube-controllers:v3.17.2",
+			CanalFlannel:              "rancher/coreos-flannel:v0.13.0-rancher1",
```

Comment:
@kinarashah on docker hub `rancher/coreos-flannel` does not have a tag `v1.15.1`. Do we have to update the image to `rancher/mirrored-coreos-flannel:v0.15.1`?

Reply:
Hey yeah, that's right - we'd be using the mirrored image `mirrored-coreos-flannel:v0.15.1` now.

Reply:
Okay thanks. Updating this now.